core/src/test/java/ai/timefold/solver/core/impl/domain/common/accessor/ReflectionBeanPropertyMemberAccessorTest.java

```diff
@@ -26,4 +28,22 @@ void methodAnnotatedEntity() throws NoSuchMethodException {
         assertThat(e1.getValue()).isSameAs(v2);
     }
 
+    @Test
```

Comment:
Is there a test for scenarios where either only the getter or only the setter method exists?

Reply:
Considering how much testdomain boilerplate would be required to test for this trivial aspect, I decided against it.

Reply:
Okay. I suggest adding a new variable to `TestdataDifferentGetterSetterVisibilityEntity` and renaming the class to something like `TestdataInvalidGetterSetterEntity`.